vita3k/renderer/src/gl/texture.cpp

```diff
@@ -216,7 +216,11 @@ void upload_bound_texture(const SceGxmTexture &gxm_texture, const MemState &mem)
             case SCE_GXM_TEXTURE_FORMAT_YUV420P2_CSC0:
             case SCE_GXM_TEXTURE_FORMAT_YVU420P2_CSC0:
             case SCE_GXM_TEXTURE_FORMAT_YUV420P2_CSC1:
-            case SCE_GXM_TEXTURE_FORMAT_YVU420P2_CSC1: {
+            case SCE_GXM_TEXTURE_FORMAT_YVU420P2_CSC1:
+            case SCE_GXM_TEXTURE_FORMAT_YUV420P3_CSC0:
+            case SCE_GXM_TEXTURE_FORMAT_YVU420P3_CSC0:
```

Comment:
Repeated?

Reply:
no, i have thinks same but in rerallity is diferent lol

Reply:
after understund, one is YUV and other is YVU :D